README.md

```diff
@@ -45,6 +45,7 @@ git push -u origin YOUR_BRANCH_NAME
 
 - [Omid Rajaei](https://www.linkedin.com/in/omid-rajaei/)
 - Matin Eshtiagh
+- Dorsa Faham
```

Comment:
You should do your changes on a new branch, not `main` branch. But this PR is ok.